website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md

```diff
@@ -0,0 +1,132 @@
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.2">
+
+- Have a multi-tenant dbt Cloud account, <a href="https://docs.getdbt.com/docs/deploy/regions">hosted</a> in North America. 
+  * Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API. <br /> 
+- Have both your production and development environments running dbt version 1.2 (latest) <br />
+- Use Snowflake data platform <br />
+- Install the <a href="https://hub.getdbt.com/dbt-labs/metrics/latest/">dbt metrics package</a> version <code>">=0.3.0", "<0.4.0"</code> in your dbt project <br />
+- Set up the <a href="https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview">Metadata API</a> in the integrated tool to import metric definitions <br />
+- Recommended - Review the <a href="https://docs.getdbt.com/docs/building-a-dbt-project/metrics">dbt metrics page</a> and <a href="https://docs.getdbt.com/blog/getting-started-with-the-dbt-semantic-layer">Getting started with the dbt Semantic Layer</a> blog <br />
+
+</VersionBlock>
+
+:::caution Considerations
+
+Some important considerations to know about during the Public Preview:
+
+- Support for Snowflake data platform only (_additional data platforms coming soon_)
+- Support for the deployment environment only (_development experience coming soon_)
+- Do not use environment variables for the job/environment (_coming soon_)
+
+:::
+
+## Public Preview
+
+The dbt Semantic Layer is currently available for Public Preview, which means:
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during Public Preview. Team and Enterprise accounts will be able to set up the Semantic Layer and [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview) in the integrated partner tool to import metric definition. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API. For more info on plan availability, check out our [product architecture](/docs/use-dbt-semantic-layer/dbt-semantic-layer#product-architecture) section.
+
+&mdash; **What?** Public Previews provide early access to new features. The Semantic Layer is stable and you can use it for production deployments, but there may still be some planned additions and modifications to product behaviors before moving to General Availability. We may also introduce new functionality that is not backwards compatible. dbt Labs provides support, and relevant service level objectives (SLOs) apply. We will introduce pricing for the dbt Semantic Layer will alongside the General Available (GA) release.
```

Comment:
There's an extra word in the last line.

> We will introduce pricing for the dbt Semantic Layer will alongside the General Available (GA) release.

I think the word `will` before the word `alongside` should be removed.